### DISCUSSION TITLE
Assign Classes with a property

### DISCUSSION THREAD

**Discussion by nielsbosma** [AvaloniaUI/Avalonia#7153] [Q&A]
Is there a better way to do this:

![image](https://user-images.githubusercontent.com/4034492/145939187-80314f74-c90e-47be-8146-515010da01c9.png)

My solution works but Ryder isn't happy about it my OCD can't stand the error marked files:

![image](https://user-images.githubusercontent.com/4034492/145939301-805cfe69-6e1d-4461-b2c5-60447775a8bc.png)

I tried with a Converter but couldn't get it to work.

**Comment by maxkatz6** [comment]
https://youtrack.jetbrains.com/issue/RIDER-65923

It's Rider's issue. Was fixed in latest release.

**Reply by nielsbosma** [reply]
Yes, ok but still would be nice to see a cleaner solution to this scenario.

**Reply by maxkatz6** [reply]
What do you mean "cleaner" solution? Syntax highlighting is provided by Rider and this issue was fixed by them.
Avalonia can compile it with or without it.

**Reply by nielsbosma** [reply]
Ok so I mean it would be nice to be able to do this:

```
...
Classes="{Binding MyStrPropWithAClassName}"
...
```
